Microsoft.Toolkit/Helpers/ObjectStorage/ISettingsStorageHelper.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Toolkit.Helpers
+{
+    /// <summary>
+    /// Service interface used to store data using key value pairs.
+    /// </summary>
+    /// <typeparam name="TKey">The type of keys to use for accessing values.</typeparam>
+    public interface ISettingsStorageHelper<TKey>
```

Comment:
Nit: the `TKey` type parameter can be declared as contravariant in this case:

```suggestion
    public interface ISettingsStorageHelper<in TKey>
```

Reply:
Good callout!